framework/config/config.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, root_dir, owtf_pid):
         # Available profiles = g -> General configuration, n -> Network plugin
         # order, w -> Web plugin order, r -> Resources file
         self.initialize_attributes()
-        self.LoadFrameworkConfigFromFile(os.path.join(self.RootDir, 'framework', 'config', 'framework_config.cfg'))
+        self.LoadFrameworkConfigFromFile(self.select_config_path(os.path.join(os.path.expanduser('~'), '.owtf', 'configuration', 'framework_config.cfg')))
```

Comment:
@DePierre Framework config is the only config file on which owtf completely depends. This file contains important parameters and location of other config files. So the path of this file has to be hard coded and fixed.